POC/pom.xml

```diff
@@ -0,0 +1,23 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
why in the POC filer? it was meant to store the original POC files, not the dev project files

Reply:
You are right, by mistake